bootstrap/bootstrap.go

```diff
@@ -291,45 +302,41 @@ func (bs *bootstrap) initialize() error {
 		//NOTE(longsleep): remove me - create keypair a random key pair.
 		sm := jwt.SigningMethodPS256
 		bs.signingMethod = sm
-		logger.WithField("alg", sm.Name).Warnf("missing --signing-private-key parameter, using random %d bit signing key", defaultSigningKeyBits)
-		signer, _ := rsa.GenerateKey(rand.Reader, defaultSigningKeyBits)
+		logger.WithField("alg", sm.Name).Warnf("missing --signing-private-key parameter, using random %d bit signing key", DefaultSigningKeyBits)
+		signer, _ := rsa.GenerateKey(rand.Reader, DefaultSigningKeyBits)
 		bs.signers[bs.signingKeyID] = signer
 	}
 
 	// Ensure we have a signer for the things we need.
-	err = validateSigners(bs)
+	err = ValidateSigners(bs)
 	if err != nil {
 		return err
 	}
 
-	validationKeysPath, _ := cmd.Flags().GetString("validation-keys-path")
-	if validationKeysPath == "" {
-		validationKeysPath = os.Getenv("KONNECTD_VALIDATION_KEYS_PATH")
-	}
+	validationKeysPath := cfg.ValidationKeysPath
 	if validationKeysPath != "" {
 		logger.WithField("path", validationKeysPath).Infoln("loading validation keys")
-		err = addValidatorsFromPath(validationKeysPath, bs)
+		err = AddValidatorsFromPath(validationKeysPath, bs)
```

Comment:
Does this need to become exported? I would prefer to keep the exported signature for stuff only relevant from connect as small as possible. The more API is exported, the more needs to be maintained. It might make sense to have this exported though.

Reply:
@longsleep Yes there are some functions which don't need to be exported, will unexport those however some functions in bootstrap/util are used by the cmd package.

![used_funckonnect](https://user-images.githubusercontent.com/246983/73734701-55211f80-473e-11ea-859c-26275ad0ad02.png)

The other approach would be to export everything in util to provide them to library consumers however your argument to keep the exported surface to a minimum also makes sense.